server/leafnode.go

```diff
@@ -869,12 +870,11 @@ func (c *client) processLeafUnsub(arg []byte) error {
 		c.Errorf(err.Error())
 	}
 	// If we are routing add to the route map for the associated account.
-	srv.updateRouteSubscriptionMap(acc, sub, -11)
```

Comment:
Not sure why this is entirely removed? I see that we still have the one with +1 in processing of Sub...

Reply:
You are correct, will add back.